cmd/image/import/import.go

```diff
@@ -191,6 +191,7 @@ func init() {
 	Cmd.Flags().StringVar(&pkg.ImageCMDOptions.ServiceCredName, "service-credential-name", "pvsadm-service-cred", "Service Credential name to be auto generated")
 
 	_ = Cmd.MarkFlagRequired("bucket")
+	_ = Cmd.MarkFlagRequired("powervs-instance-name")
 	_ = Cmd.MarkFlagRequired("image-name")
 	_ = Cmd.MarkFlagRequired("object-name")
 	_ = Cmd.MarkFlagRequired("region")
```

Comment:
This "region" is cos region or pvs region? Looking at the code it can be inferred. But lets make it unambiguous for end-users